config/brokers.toml

```diff
@@ -14,12 +14,14 @@ private_key = ""
 [ib]
 host = "127.0.0.1"
 
-[ib.accounts]
-margin = ""
-registered = ""
-paper = ""
+ports.gw = 4002
+ports.tws = 7497
+ports.order = ["gw", "tws",]
 
-[ib.ports]
-gw = 4002
-tws = 7497
-order = [ "gw", "tws",]
+accounts.margin = "X0000000"
```

Comment:
So basically all providers can define a table like this now.

We might tweak the schema a bit but i think this is simple enough to start.